index.md

```diff
@@ -4,26 +4,24 @@ sidebar: home_sidebar
 layout: page
 ---
 
-Flutter is a new project to help developers build high-performance, high-fidelity,
-mobile apps for iOS and Android from a single codebase.
-
 {% include note.html content="_Flutter is an early-stage open-source project_. We hope you try Flutter out and send us [feedback](mailto:flutter-dev@googlegroups.com)." %}
 
 ## What is Flutter?
 
-Flutter provides an easy and productive way to build and deploy high-performance
-mobile apps for Android and iOS. The goal is to deliver apps that feel natural
-on different platforms, and we embrace differences in scrolling behaviors,
-typography, icons, and more.
+Flutter is a mobile app SDK for building high-performance, high-fidelity, 
+apps for iOS and Android, from a single codebase.
+
+The goal is to enable developers to deliver high-performance apps that feel natural 
+on different platforms. We embrace differences in scrolling behaviors, 
+typography, icons, and more. 
 
 ![Shrine, iOS left and Android right](/images/whatisflutter/shrine_both.png)
 
-This is an online marketplace app from the [Gallery](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery/lib/demo),
-a showcase of Flutter apps to download and run locally. Shrine has
-high-quality scrolling images, interactive cards, buttons, dropdown lists,
-and a shopping cart page. You can run Shrine on an emulator or a phone, then check
-out the single codebase for this and more examples [here](https://github.com/flutter/flutter/tree/master/examples)
-in our GitHub repository.
+This is a demo app from the [Gallery](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery/lib/demo),
+a collection of Flutter sample apps you can run after installing Flutter and setting
```

Comment:
The term 'Flutter sample apps' is a little confusing; this is a single iOS/Android app